cluster-autoscaler/clusterstate/utils/types.go

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
I was rather thinking about putting log to api. But we don't have to - it can stay here but please combine logging.go and types.go - there is no reason to keep them separate now.